### DISCUSSION TITLE
Duration of recorded videos is more than 10 seconds.

### DISCUSSION THREAD

**Discussion by 210392** [blakeblackshear/frigate#14976] [Ask A Question] [answered]
Hello.
Tell me, can I increase the duration of the recorded videos to more than 10 seconds?
Frigate version 0.15.0

**Comment by hawkeye217** [comment, marked as answer]
No, you cannot. Why do you want to do that?

**Reply by 210392** [reply]
I use the frigate as an addition to Homeassistant. I created an automation that sends me a video in Telegram when creating a video. I get a lot of 10 second videos.

**Reply by hawkeye217** [reply]
You should use the clip endpoint from the HTTP API for that: https://docs.frigate.video/integrations/api#get-apicamera_namelabelclipmp4

**Reply by 210392** [reply]
> Для этого вам следует использовать конечную точку клипа из HTTP API: https://docs.frigate.video/integrations/api#get-apicamera_namelabelclipmp4
Could you show an example of such automation?

**Reply by hawkeye217** [reply]
See this tip: https://github.com/blakeblackshear/frigate/discussions/2898
